src/components/DocumentationTopic.vue

```diff
@@ -423,6 +425,7 @@ export default {
   },
   data() {
     return {
+      appState: AppStore.state,
```

Comment:
Can you copy it? GitHub is linking between a couple different messages, and I can't tell which one is which.

Reply:
This is fairly urgent since it's a performance regression. We'll need to resolve the conflicts in whichever PR is merged last, but I expect it to be pretty simple since this array is only referenced a few times in the source and will just be moving to a new store.